modules/machineset-manually-scaling.adoc

```diff
@@ -34,20 +34,3 @@ $ oc edit <machineset> -n openshift-machine-api
 +
 You can scale the MachineSet up or down. It takes several minutes for the new
 machines to be available.
-+
-[IMPORTANT]
-====
-The default installation creates only one router. If you scale down your cluster,
```

Comment:
Some facts:
1. currently two router pod will be installed by default.
2. router pods are deployed to `worker` node.
3. if you scale down your cluster and no worker nodes, then router pod should be in `pending` status and cannot access the web console.

Reply:
my suggestion as below and please help to refine them
```
[IMPORTANT]
====
The default installation creates two routers and deploys them on worker nodes. If you 
scale down your cluster, please ensure at least one worker node is running. otherwise, 
you may cannot access the web console and some other resources. 

You can check the status of the router pod by running:
----
$ oc get pod -n openshift-ingress
----